src/main/webapp/app/exam/participate/exercises/programming/programming-exam-submission.component.ts

```diff
@@ -103,11 +103,17 @@ export class ProgrammingExamSubmissionComponent extends ExamSubmissionComponent
     }
 
     onActivate(): void {
+        super.onActivate();
         this.reload();
     }
 
     updateSubmissionFromView(): void {
         // Note: we just save here and do not commit, because this can lead to problems!
         this.codeEditorContainer.actions.onSave();
     }
+
+    updateViewFromSubmission(): void {
+        // Note: we just save here and do not commit, because this can lead to problems!
+        this.codeEditorContainer.actions.onSave();
```

Comment:
I think we could just leave this method empty and write a comment about it. Semantically it does not make sense to call `onSave` when we want to update the view from the submission (not needed for programming exercises anyways).